specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-03-01-preview/FlexibleServers.json

```diff
@@ -0,0 +1,1087 @@
+      "properties": {
+        "storageSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Max storage allowed for a server."
+        },
+        "autoGrow": {
+          "type": "string",
+          "description": "Flag to enable / disable Storage Auto grow for flexible server.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "StorageAutoGrow",
+            "modelAsString": true
+          }
+        },
+        "iopsTier": {
+          "type": "string",
+          "description": "Name of storage tier for IOPS.",
+          "enum": [
+            "P1",
+            "P2",
+            "P3",
+            "P4",
+            "P6",
+            "P10",
+            "P15",
+            "P20",
+            "P30",
+            "P40",
+            "P50",
+            "P60",
+            "P70",
+            "P80"
+          ],
+          "x-ms-enum": {
+            "name": "AzureManagedDiskPerformanceTiers",
+            "modelAsString": true
+          }
+        },
+        "iops": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Storage tier IOPS quantity.",
+          "readOnly": true
+        }
+      },
+      "description": "Storage properties of a server"
+    },
+    "AuthConfig": {
+      "type": "object",
+      "properties": {
+        "activeDirectoryAuth": {
+          "type": "string",
+          "description": "If Enabled, Azure Active Directory authentication is enabled.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "activeDirectoryAuthEnum",
+            "modelAsString": true
+          }
+        },
+        "passwordAuth": {
+          "type": "string",
+          "default": "Enabled",
+          "description": "If Enabled, Password authentication is enabled.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "passwordAuthEnum",
+            "modelAsString": true
+          }
+        },
+        "tenantId": {
+          "type": "string",
+          "default": "",
+          "description": "Tenant id of the server."
+        }
+      },
+      "description": "Authentication configuration properties of a server"
+    },
+    "DataEncryption": {
+      "type": "object",
+      "properties": {
+        "primaryKeyURI": {
+          "type": "string",
+          "description": "URI for the key for data encryption for primary server."
+        },
+        "primaryUserAssignedIdentityId": {
+          "type": "string",
+          "description": "Resource Id for the User assigned identity to be used for data encryption for primary server."
+        },
+        "geoBackupKeyURI": {
+          "type": "string",
+          "description": "URI for the key for data encryption for geo-backup of server."
+        },
```

Comment:
What are valid values? Is this URI supposed to be a reference to a customer key in keyvault or something?